packages/config/config/devices/0x031e/lzw45.json

```diff
@@ -0,0 +1,1169 @@
+{
+	"manufacturer": "Inovelli",
+	"manufacturerId": "0x031e",
+	"label": "LZW45",
+	"description": "Light Strip",
+	"devices": [
+		{
+			"productType": "0x000a",
+			"productId": "0x0001"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 5,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Basic_Set",
+			"maxNodes": 5
+		},
+		"3": {
+			"label": "Switch_MultiLevel_Set",
+			"maxNodes": 5
+		},
+		"4": {
+			"label": "Switch_MultiLevel_Start/Stop",
+			"maxNodes": 5
+		}
+	},
+	"paramInformation": {
+		"1": {
+			"label": "Number of Pixels",
+			"description": "When individually addressable LEDs are used, this parameter tells the controller the number of pixels that are attached.  0: Automatic recognition of pixels. Range 1 - 130: Set the fixed value of the pixel bit. Default: 0",
```

Comment:
The description should not have default values and ranges and such. Special meanings (e.g. off) should be conveyed with an `"option"` where it makes sense. Same goes for the other params.